### ISSUE TITLE
[BUG]: Notification Tab error

### ISSUE THREAD

**Issue by nascarrocks** [yuliskov/SmartTube#4095]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

24.61

### Device Type

Smart TV/Box

### Affected Android

Firestick OS

### Steps to reproduce the bug

Go to notifications and click it 

### Actual behavior

It comes up saying can't load content 
![17313744433881352335441619990986](https://github.com/user-attachments/assets/9eb1b149-acdf-4f06-a5a8-0d8927872390)

### Additional information

I'm running it on a Firestick 4k

**Comment by Peachy1177** [comment]
I'm having this EXACT same issue.  I have a 4K Max Firestick that I'm running it on.  It started about two weeks ago and I can't figure out how to fix it.  I tried messing around with the settings.  I uninstalled the app and reinstalled it, and nothing seems to work.  I'm assuming it must be a bug, but seeing that there is a lack of information anywhere online (this was the first place I found someone who had a similar issue) I don't know how common this issue is for other people.

**Comment by lilchina** [comment]
I get Error Response: request contains  an invalid  argument. I am on version  24.71 updated  11/4/2024

**Comment by Peachy1177** [comment]
I'm not home at the moment, so I'm not sure which version I'm on, but I know when I checked the other day it was the most recent version.  Like I mentioned, I tried looking through the settings, but I know I hadn't touched my settings prior to this happening all of a sudden, so I don't know what caused it.  I can't imagine you and I are the only ones having this issue.

**Comment by larryvgs** [comment]
Just updated to latest 24.71 and have the same problem on my Toshiba Fire TV (Fire OS7).  Notifications are signed out.  Everything else, history and subscriptions are ok.  Signed in and out multiple times and reinstalled.  Same problem.

**Comment by Peachy1177** [comment]
Same here.  I updated last night...hopeful it would have a fix for the issue.  The error message is different on the notification tab now, instead of the one that you posted above (which is what I was seeing too before), now it says, "ErrorResponse: Request contains an invalid argument." And underneath it says to sign in, but I am signed in.  I don't know what the issue is all of a sudden with this app and notifications, and I can't imagine we are the only two people having this problem.

**Comment by larryvgs** [comment]
Yup.  Just uninstalled, reinstalled, and signed in again and now am getting the same error message as you under Notifications.

**Comment by lilchina** [comment]
I've done the same complete uninstall, re-install. After new install I login and it  has my 2 profiles.  Also because this is a fresh install notification i not on the left so I add it  back  and it  has a login prompt.  So I  login again an choose my default profile not the secondary. It immediately  get   the error and will ask for a login again. Not complaining just giving you  my experience.  was working a few days back so something has  changed on YT end so that now Notification do not show my shows that I ask for notifications and errors out  and wants a login.

**Comment by Peachy1177** [comment]
I don't know if YouTube did something to screw with 3rd party apps, but it seems odd that notifications would be the only thing affected.  I really had hoped the update yesterday would fix things, but it just changed the error message from one thing to another.  Well, if either of you come up with a solution, keep me posted because this is driving me nuts.

**Comment by Red-Baran** [comment]
Same error on Google streamer

**Comment by patrik-malina** [comment]
Still not working. On tcl tv with ST 25.64
